.changeset/chilly-buttons-agree.md

```diff
@@ -0,0 +1,19 @@
+---
+"@salt-ds/theme": patch
+---
+
+Deprecated the following size tokens:
```

Comment:
@pseys we used to have a page on storybook that also contained the deprecated tokens. I'm guessing we want that on the site too

Reply:
should this PR also add the new token to the size page on the site?

Reply:
Good shout